speakers/src/test/java/in/droidcon/speakers/domain/GetAllSpeakersTest.kt

```diff
@@ -0,0 +1,44 @@
+package `in`.droidcon.speakers.domain
```

Comment:
Same here, please create separate PR for Speaker module / speaker detail module related changes.

Reply:
Same scenario - the problem is, There are a lot of refactoring for speakers module. I have moved epoxy and DTOs to base for this module, so that team, sponsors and speakers + sponsor detail, speaker-detail and team can reuse stuff. It's better to keep it incremental. As more refactoring might come if we work on sponsors.